pages/api/auth/[...nextauth].js

```diff
@@ -1,15 +1,16 @@
 import NextAuth from 'next-auth';
-import GithubProvider from 'next-auth/providers/github';
```

Comment:
Right now the flow is that the teachers sign in with their Github account. 

From my knowledge we only want freeCodeCamp students to sign in with auth0 and teachers will sign in with their Github Account (this can be extended to sign in with Google, Facebook, Email, whatever) to have an admin type of account.

Whenever you sign in with Github, someone is added to the `User` table (the teacher table) so swapping in Auth0 like this will mean only teachers will be able to sign into classroom with their FCC accounts.

We need to see if NextAuth supports two types of accounts with different levels of access: Student (can't see much) and Teacher (ability to create classrooms, add people to classrooms) accounts.

Reply:
https://www.reddit.com/r/nextjs/comments/udsje3/can_nextauthjs_have_multiple_auth_routes/ Reddit thread where the user has a similar problem.

Reply:
Thanks, Rob.  

IMHO, you are confusing authentication and authorization. They are related but two different things. I really wanted to discuss this in #72, so I will leave comments on that issue for posterity and elaborate there.